### PR TITLE
Edidt readme

### PR DESCRIPTION
**Pull request by mayabrandi**
Adding a section for trouble shooting

**Review:**
- [ ] code approved by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,3 +129,47 @@ SQLALCHEMY_DATABASE_URI=mysql+pymysql://remoteuser:<password>@127.0.0.1:<port>/c
...
+
+## Trouble shooting
+
+When a script is failing, usually as a developer, you will get this information from the lims user who has run the script from within a specific lims step. Its easiest to trouble shoot if the step is still opened.
+
+![](img/debug_step.png)
+
+**Trouble shooting - step by step:**
+* Ask the user to keep the step opened for you to trouble shoot, if possible. (Sometimes they need to continue the step)
+* Go to the step to see what EPP was failing. The name of the blue buttom. In this case: **1. Copy UDFs from AggregateQC - Twist**
```
Comment (henrikstranneheim) [inline README.md:141]:
```suggestion
* Go to the step to see what EPP was failing. The name of the EPP is the label on blue button. In this case: **1. Copy UDFs from AggregateQC - Twist**
```

File: README.md [hunk 2]
```diff
@@ -129,3 +129,47 @@ SQLALCHEMY_DATABASE_URI=mysql+pymysql://remoteuser:<password>@127.0.0.1:<port>/c
...
+* One way to debug is to run the script from command line. ssh into productuoin as described above and run the script with the same argument that are given in the "command line" text box. The process id {processLuid} is allmost allways asked for. 
+
+![](img/debug_automation.png)
+
+
+`{processLuid} = <magic number>-<the last section of the url of the step>` 
+
+In this case: 24-144356. 
+  
+The magic number can be: 24, 151 or 122.
```
Comment (henrikstranneheim) [inline README.md:153]: What is the magic number? And what does it do (accept for being magic...)

Comment (mayabrandi) [reply README.md:153]: It is just a wierd prefix of the process ID. I never managed to find out how its picked and why. I contacted genologics to get the logics behind it. Waiting for reply.

Comment (henrikstranneheim) [reply README.md:153]: 👍

**Pull request merged**